libtextsecure/account_manager.js

```diff
@@ -537,6 +537,7 @@
       // Update the conversation if we have it
       const newProfile = storage.getLocalProfile();
       await conversation.setProfile(newProfile);
+      await conversation.setFriendRequestStatus(window.friends.friendRequestStatusEnum.friends);
```

Comment:
Wouldn't it be better to do this in the conversation init?
```
if (me && friendRequestStatus != friends) {
    setFriendRequestStatus(friends)
}
```

This would cover the case where the conversation somehow gets deleted after registration

Reply:
Tru